src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -1,654 +1,670 @@
      */
-    private static final int BASE_MAX_ATTACKS = 25;
+    private static final double PITCH_MULTIPLIER = 0.4D;
 
     /**
-     * Y range in which the guard detects other entities.
+     * Ranged attack velocity
      */
-    private static final double HEIGHT_DETECTION_RANGE = 10D;
+    private static final float RANGED_VELOCITY = (float) 1.6D;
 
     /**
-     * Max tries to find a position to path to.
+     * Quantity the worker should turn around all at once.
      */
-    private static final int MAX_TRIES = 20;
+    private static final double TURN_AROUND = 180D;
 
     /**
-     * Path that close to the patrol target.
+     * Normal volume at which sounds are played at.
      */
-    private static final int PATH_CLOSE = 2;
+    private static final double BASIC_VOLUME = 1.0D;
 
     /**
-     * The dump base of actions, will increase depending on level.
+     * Experience to add when a mob is killed
      */
-    private static final int DUMP_BASE = 5;
+    private static final int EXP_PER_MOD_DEATH = 5;
 
     /**
-     * Increases the max attacks by this amount per level.
+     * How many ticks are in a second
      */
-    private static final int ADDITIONAL_MAX_ATTACKS_PER_LEVEL = 5;
+    private static final int TICKS_PER_SECOND = 20;
 
     /**
-     * After this amount of ticks the citizen should switch the patrolling target because his current one is unreachable.
+     * Base physical damage.
      */
-    private static final int SWITCH_TARGET_AFTER_TICKS = 10;
+    private static final int BASE_PHYSICAL_DAMAGE = 3;
 
     /**
-     * Low health at which the guards should retrieve.
+     * Tools and Items needed by the worker.
      */
-    private static final double LOW_HEALTH = 2.0;
+    public final List<ToolType>  toolsNeeded = new ArrayList<>();
+    public final List<ItemStack> itemsNeeded = new ArrayList<>();
 
     /**
-     * The length range one patrolling operation can have on x or z.
+     * How many more ticks we have until next attack.
      */
-    private static final int LENGTH_RANGE = 10;
+    protected int currentAttackDelay = 0;
 
     /**
-     * The length range one patrolling operation can have on x or z.
+     * Physical Attack delay in ticks.
      */
-    private static final int IN_TEN = 10;
+    protected static final int RANGED_ATTACK_DELAY_BASE = 20;
 
     /**
-     * The length range one patrolling operation can have on y.
+     * Ranged hit chance devider.
      */
-    private static final int UP_DOWN_RANGE = 4;
+    private static final double HIT_CHANCE_DIVIDER = 15.0D;
 
     /**
-     * Chance of trying to follow the same direction again.
+     * Have to aim that bit higher to hit the target.
      */
-    private static final int LAST_DIRECTION_CHANCE = 5;
+    private static final double RANGED_AIM_SLIGHTLY_HIGHER_MULTIPLIER = 0.20000000298023224D;
 
     /**
-     * The current target.
+     * Quantity to be moved to rotate the entity without actually moving.
```

Comment:
Can we put those constants into an extra class to improve readability?

Reply:
Yes. I could probably clean them up in general